.github/workflows/test-integration-runner.yaml

```diff
@@ -439,6 +443,21 @@ jobs:
           envsubst < $V > $V.bk
           cat $V.bk
           mv $V.bk $V
+          echo "################################################################################"
+
+          echo "Values file: $V"
```

Comment:
Using those short names doesn't help at all and doesn't add much value.
Let's use a clear variable name here.